webapp/components/create_post.jsx

```diff
@@ -698,6 +766,14 @@ export default class CreatePost extends React.Component {
                     show={this.state.showPostDeletedModal}
                     onHide={this.hidePostDeletedModal}
                 />
+                <ConfirmModal
+                    title={notifyAllTitle}
+                    message={notifyAllMessage}
+                    confirmButton={notifyAllConfirm}
+                    show={this.state.showConfirmModal}
+                    onConfirm={this.handleNotifyAllConfirmation}
+                    onCancel={() => this.setState({showConfirmModal: false})}
```

Comment:
He means make a function as part of the component and pass that to `onCancel`, like this

```
class CreatePost extends React.Component {
  handleCancel = () => { // do stuff }
  render() {
    ...
    onCancel={this.handleCancel}
    ...
  }
}
```